database/migrations/new/2023_02_14_085525_update_recommendation_table.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Support\Facades\DB;
+
+class UpdateRecommendationTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+
+    public function __construct()
+    {
+        DB::getDoctrineSchemaManager()->getDatabasePlatform()->registerDoctrineTypeMapping('enum', 'string');
+    }
+    public function up()
+    {
+        Schema::table('recommendations', function (Blueprint $table) {
+            //
+            $table->string('type')->change();
```

Comment:
What is it before?

Reply:
enum(formal, informal) and we werent using it